src/database/database-internal.ts

```diff
@@ -89,7 +89,8 @@ export class DatabaseService {
 
     let db: Database = this.databases[dbUrl];
     if (typeof db === 'undefined') {
-      const rtdb = require('@firebase/database'); // eslint-disable-line @typescript-eslint/no-var-requires
+      // eslint-disable-next-line @typescript-eslint/no-var-requires
+      const rtdb = require('@firebase/database-compat/standalone');
```

Comment:
Can you just import `initStandalone` at the top of the file instead of using require here?

Reply:
I believe this was necessary to defer loading RTDB dependencies until needed. The `database-internal` module is imported by the `firebase-app` module, which gets loaded pretty early on during `initializeApp()`. So we had to defer importing any values from RTDB till later. Looking at the latest code, however, I think it's safe to change this. I see that `firebase-app` now only imports a type from `database-internal`. But I'd prefer to not make any such changes at this point.

Once we switched to our modularized implementation, this will be lot easier to implement.